contracts/smart-account/modules/AccountRecoveryModule.sol

```diff
@@ -378,6 +306,27 @@ contract AccountRecoveryModule is
         );
     }
 
+    function resetModuleForCaller(bytes32[] memory guardians) external {
```

Comment:
how would one procure guarding hashes?

Reply:
ask a guardian (person) to sign the `CONTROL_MESSAGE`, then hash the signature